src/System.Text.Json/src/System/Text/Json/Node/JsonString.cs

```diff
@@ -38,16 +40,28 @@ public sealed class JsonString : JsonNode, IEquatable<JsonString>
         public JsonString(Guid value) => Value = value.ToString();
 
         /// <summary>
-        ///  Initializes a new instance of the <see cref="JsonString"/> with a string representation of the <see cref="DateTime"/> structure.
+        ///  Initializes a new instance of the <see cref="JsonString"/> with an ISO 8601 representation of the <see cref="DateTime"/> structure.
         /// </summary>
         /// <param name="value">The value to represent as a JSON string.</param>
-        public JsonString(DateTime value) => Value = value.ToString();
+        /// <exception cref="FormatException">
+        ///   Text value of this instance is not in an ISO 8601 defined DateTimeOffset format.
+        /// </exception>
+        /// <exception cref="ArgumentOutOfRangeException">
+        ///   The date and time is outside the range of dates supported by the calendar used by the invariant culture.
+        /// </exception>
+        public JsonString(DateTime value) => Value = value.ToString("s", CultureInfo.InvariantCulture);
 
         /// <summary>
-        ///  Initializes a new instance of the <see cref="JsonString"/> with a string representation of the <see cref="DateTimeOffset"/> structure.
+        ///  Initializes a new instance of the <see cref="JsonString"/> with an ISO 8601 representation of the <see cref="DateTimeOffset"/> structure.
         /// </summary>
         /// <param name="value">The value to represent as a JSON string.</param>
-        public JsonString(DateTimeOffset value) => Value = value.ToString();
+        /// <exception cref="FormatException">
+        ///   Text value of this instance is not in an ISO 8601 defined DateTimeOffset format.
```

Comment:
Why is the ctor throwing a FormatException?

Reply:
Right, it shouldn't be.